### ISSUE TITLE
Some questions about the SensatUrban dataset

### ISSUE THREAD

**Issue by 1kongkong** [shenglandu/PushBoundary#7]
Hello, thank you very much for the code you provided. I have a few questions regarding the SensatUrban dataset.
1. Is it possible to obtain the ground truth for the current test set? If the ground truth isn't publicly available, how can we assess the model's accuracy on the test set?
2. In the training set, there are two areas with very small point cloud data (cambridge_block_0 & 1). Could this issue, possibly caused by data corruption, have been resolved?

**Comment by shenglandu** [comment]
> Hello, thank you very much for the code you provided. I have a few questions regarding the SensatUrban dataset.
> 
> 1. Is it possible to obtain the ground truth for the current test set? If the ground truth isn't publicly available, how can we assess the model's accuracy on the test set?
> 2. In the training set, there are two areas with very small point cloud data (cambridge_block_0 & 1). Could this issue, possibly caused by data corruption, have been resolved?

Hi there,

Thanks for your interest in this work! 
(1) When I wrote this paper, I didn't have access to the GT labels. The performance of my method was measured via the official [evaluation portal](https://competitions.codalab.org/competitions/31519#participate-submit_results).

(2) Yes, there are two very small sets that caused some troubles also for my network. During training, I simply ignored them. Otherwise, they could lead to very sparsely sampled points in a batch, thus making the network learning unstable. I don't have a very good idea how this issue can be resolved. Maybe you can turn to the author of SensatUrban and he can share with you his ideas.


Best,
Shenglan

**Comment by 1kongkong** [comment]
Your reply has been very helpful to me, thank you very much!

**Comment by 1kongkong** [comment]
> > Hello, thank you very much for the code you provided. I have a few questions regarding the SensatUrban dataset.
> > 
> > 1. Is it possible to obtain the ground truth for the current test set? If the ground truth isn't publicly available, how can we assess the model's accuracy on the test set?
> > 2. In the training set, there are two areas with very small point cloud data (cambridge_block_0 & 1). Could this issue, possibly caused by data corruption, have been resolved?
> 
> Hi there,
> 
> Thanks for your interest in this work! (1) When I wrote this paper, I didn't have access to the GT labels. The performance of my method was measured via the official [evaluation portal](https://competitions.codalab.org/competitions/31519#participate-submit_results).
> 
> (2) Yes, there are two very small sets that caused some troubles also for my network. During training, I simply ignored them. Otherwise, they could lead to very sparsely sampled points in a batch, thus making the network learning unstable. I don't have a very good idea how this issue can be resolved. Maybe you can turn to the author of SensatUrban and he can share with you his ideas.
> 
> Best, Shenglan

Your reply has been very helpful to me, thank you very much!

**Comment by shenglandu** [comment]
> > > Hello, thank you very much for the code you provided. I have a few questions regarding the SensatUrban dataset.
> > > 
> > > 1. Is it possible to obtain the ground truth for the current test set? If the ground truth isn't publicly available, how can we assess the model's accuracy on the test set?
> > > 2. In the training set, there are two areas with very small point cloud data (cambridge_block_0 & 1). Could this issue, possibly caused by data corruption, have been resolved?
> > 
> > 
> > Hi there,
> > Thanks for your interest in this work! (1) When I wrote this paper, I didn't have access to the GT labels. The performance of my method was measured via the official [evaluation portal](https://competitions.codalab.org/competitions/31519#participate-submit_results).
> > (2) Yes, there are two very small sets that caused some troubles also for my network. During training, I simply ignored them. Otherwise, they could lead to very sparsely sampled points in a batch, thus making the network learning unstable. I don't have a very good idea how this issue can be resolved. Maybe you can turn to the author of SensatUrban and he can share with you his ideas.
> > Best, Shenglan
> 
> Your reply has been very helpful to me, thank you very much!

Hi there,

No worries! :)

Best,
Shenglan

**Comment by 1kongkong** [comment]
Hello!
I hope this message finds you well. 

Following the method you suggested, I have registered using my university email to participate in the competition organized by the authors. However, my participation eligibility is currently under review with the status message: "Your request to participate in this challenge has been received and a decision is pending."

Considering your experience with participating in such competitions, I am curious about the approximate duration of the verification process. Are there any alternative methods or steps that might expedite the verification process?

Your guidance or insights on this matter would be greatly appreciated.

**Comment by shenglandu** [comment]
> Hello! I hope this message finds you well.
> 
> Following the method you suggested, I have registered using my university email to participate in the competition organized by the authors. However, my participation eligibility is currently under review with the status message: "Your request to participate in this challenge has been received and a decision is pending."
> 
> Considering your experience with participating in such competitions, I am curious about the approximate duration of the verification process. Are there any alternative methods or steps that might expedite the verification process?
> 
> Your guidance or insights on this matter would be greatly appreciated.

Hello,

For me, it took around 3-5 working days to wait during pending status and get the access to the competition. I am not clear how the verification can be speeded up as it is done by the organizers. Maybe you can email the organizer to see if it's possible to quickly verify it? Good luck with participating the competition. 

Best,
Shenglan

**Comment by 1kongkong** [comment]
> > Hello! I hope this message finds you well.
> > Following the method you suggested, I have registered using my university email to participate in the competition organized by the authors. However, my participation eligibility is currently under review with the status message: "Your request to participate in this challenge has been received and a decision is pending."
> > Considering your experience with participating in such competitions, I am curious about the approximate duration of the verification process. Are there any alternative methods or steps that might expedite the verification process?
> > Your guidance or insights on this matter would be greatly appreciated.
> 
> Hello,
> 
> For me, it took around 3-5 working days to wait during pending status and get the access to the competition. I am not clear how the verification can be speeded up as it is done by the organizers. Maybe you can email the organizer to see if it's possible to quickly verify it? Good luck with participating the competition.
> 
> Best, Shenglan

Thank you very much for your helpful reply~
